src/styles/blocks/card.scss

```diff
@@ -0,0 +1,54 @@
+.card {
+  box-sizing: border-box;
+  position: relative;
+  text-align: justify;
+  width: 200px;
+  height: 407px;
+  border: 1px solid $grey-star-color;
+  border-radius: 5px;
+}
+
+.card__product {
+  display: block;
+  padding-inline: $standart-inline-pad;
+  padding-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: $standart-black-color;
+  text-decoration: none;
+}
+
+.card__code {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding-inline: $standart-inline-pad;
+  padding-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-inline: 16px;
+  height: 40px;
+  border: 1px solid $grey-star-color;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  text-decoration: none;
+  color: #fff;
```

Comment:
Ensure that the color value is defined as a variable in a separate file, as per the task requirements for maintainability.